src/app/hooks/useViewTracker/index.tsx

```diff
@@ -32,13 +34,18 @@ const getComponentViewTracker = (eventTrackingData?: EventTrackingData) => {
     statsDestination,
     campaignID,
     detailedPlacement,
-    optimizely,
+    sendOptimizelyEvents,
```

Comment:
Added this flag as an aid to using the OptimizelyContext directly in this hook. We don't want to always enable Optimizely for view/click events, only per component.